### PR TITLE
home page content updated with a nice button

### PR DESCRIPTION
**Pull request by yannklein**
@dmbf29 check that out
image

### CODE REVIEW

**Comment by dmbf29** [comment]
not great but it's a start

**Comment by dmbf29** [comment]
do you really like that blue for the button?

**Comment by dmbf29** [comment]
marginally better but i dont know what i expected from you

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="container">
   <h1>Welcome everyone!!</h1>
   <p>Easy Rails on Kubernetes approach for Digital Ocean and a classic Rails stack: Puma, Redis, Sidekiq, Postgres, Action Cable, Webpacker. Helm 3 in production and docker-compose + dip (https://github…</p>
-  <%= link_to "Check us out!", "#", class:"btn btn-primary" %>
+  <%= link_to "Check us out!", "#", class:"btn btn-secondary" %>
```
Comment (yannklein) [inline app/views/pages/home.html.erb:4]: You love that @dmbf29 ??

Comment (dmbf29) [reply app/views/pages/home.html.erb:4]: love, no. but we need to ship it ASAP

**Pull request merged**